tests/test_compressors/test_model_compressor.py

```diff
@@ -16,11 +16,24 @@
 
 import pytest
 from compressed_tensors.compressors.model_compressor import ModelCompressor
+from compressed_tensors.config.base import SparsityCompressionConfig
+from compressed_tensors.quantization.quant_config import QuantizationConfig
+
+
+def compressed_tensors_config_available():
```

Comment:
nit make private

Reply:
Moved to `testing_utils.py`, which contains functions to be used publicly